source/extensions/filters/http/cors/cors_filter.cc

```diff
@@ -200,11 +205,37 @@ bool CorsFilter::allowCredentials() {
   return false;
 }
 
+absl::optional<std::string> CorsFilter::runtimeKey(const std::string& key) {
```

Comment:
It doesn't seems you use `nullopt` and assume this returning `nullopt`, just make it string?

Reply:
yeah I was just updating to return `nullopt`. @venilnoronha suggested this change. I was originally returning an empty string.

Should I continue down the path of `optional` or revert back to a string?

Reply:
Sorry about that, and thanks!